### ISSUE TITLE
Sigpatches link broken

### ISSUE THREAD

**Issue by JamesDoesGaming902** [HamletDuFromage/aio-switch-updater#211]
**Describe the bug**
> Sigpatches has a broken link in the app at the moment. When going to then sigpatches tab, it says it couldn't find a link

**To Reproduce**
> Not sure if its only for me, but I just opened aio and the link was broken

**Screenshots/Crash code/Crash reports**
> I dont have access to my switch atm sorry

**System Firmware Version and CFW Version**
>  EmuNAND: 14.1.2
> Atmosphere: 1.3.2

**SD card format**
> FAT32

**Additional context**
> Pretty sure it is because sigpatches was taken down and has now moved to sigmapatches.coomer.party

**Comment by viktortrass** [comment]
yes, it is still not working

**Comment by HamletDuFromage** [comment]
I've decided to stop providing sigpatches. The "sigpatches" tab has been replaced by a "custom download" tab

**Comment by DraconicNEO** [comment]
> I've decided to stop providing sigpatches. The "sigpatches" tab has been replaced by a "custom download" tab

Why? Why not just update the link to use the ones from Jits or sigmapatches?

**Comment by HamletDuFromage** [comment]
> > I've decided to stop providing sigpatches. The "sigpatches" tab has been replaced by a "custom download" tab
> 
> Why? Why not just update the link to use the ones from Jits or sigmapatches?

With the DMCA, Nintendo made it clear that they don't want those to be distributed

**Comment by DraconicNEO** [comment]
> > > I've decided to stop providing sigpatches. The "sigpatches" tab has been replaced by a "custom download" tab
> > 
> > 
> > Why? Why not just update the link to use the ones from Jits or sigmapatches?
> 
> With the DMCA, Nintendo made it clear that they don't want those to be distributed

I don't think that's a good reason, Nintendo submits fraudulent takedown requests all the time, they especially do it on YouTube videos but that doesn't mean people just don't make videos on the Nintendo switch. 

Their takedown requests don't have much meaning, hell if they really wanted to do this project in they could probably have it taken down for the firmware update capability, or maybe even the cheat downloading capability.

This is not the kind of thing to get squeamish about, it's actually probably one of the least dangerous things considering signature patches don't actually contain copyrighted code. If you're that afraid you probably shouldn't be offering the ability to update firmware either, those do contain copyrighted code. You could argue that you're not actually including it in the app but you're not including signature patches in the app either.

**Comment by HamletDuFromage** [comment]
@DraconicNEO you're right, I would remove the option to download firmwares (that are offered by Nintendo for free) should Nintendo DMCA archive.org where I source them. There is a disclaimer in my readme stating as much

I don't want my repo to be taken down. If you're braver than I am, feel free to maintain a fork, all the aspects of the code is Free Software

**Comment by DraconicNEO** [comment]
> @DraconicNEO you're right, I would remove the option to download firmwares (that are offered by Nintendo for free) should Nintendo DMCA archive.org where I source them. There is a disclaimer in my readme stating as much

Good thing it's not likely to happen, it is notoriously difficult to remove content from archive.org, I should know I uploaded a few mainstream movies on there and they're still up to this day.
(Note: these are rare and failed movies, very difficult to find DVD copies and essentially non-existent on streaming services. I uploaded them there is a matter of preservation since I'm pretty certain that they won't survive in any other form.)

> I don't want my repo to be taken down. If you're braver than I am, feel free to maintain a fork, all the aspects of the code is Free Software

I'll probably do that, eventually. I do think it's funny how me, a furry who's Fursona is a version of Toothless from httyd is braver than the guy with GigaChad as his profile picture.

I hope you don't take up too much offense to that, I'm not trying to be mean I just think it's kind of funny.
